lib/pbench/server/api/__init__.py

```diff
@@ -207,8 +201,7 @@ def create_app(server_config):
 
```

Comment:
Line 199 looks like
```
    app = Flask("api-server")
```
but the [Flask documentation](https://flask.palletsprojects.com/en/latest/api/#flask.Flask) recommends something more like this:
```
app = Flask(__name__.split('.')[0])
```
(The docs mention that this enables the Flask-SQLAlchemy extension, which I think we use.)